CHANGELOG.md

```diff
@@ -4,10 +4,15 @@ All notable changes to the "vscode-java-test" extension will be documented in th
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## 0.14.0
+## 0.14.0 - [Unreleased]
 
 ## Added
 - Add the log level setting. [#555](https://github.com/Microsoft/vscode-java-test/issues/555)
+- Navigate to the source code from the test explorer. [#558](https://github.com/Microsoft/vscode-java-test/issues/558)
+
+## Changed
+- Redesign the user experience of running tests with configurations. [#524](https://github.com/Microsoft/vscode-java-test/issues/524)
+  > More details can be found [here](https://aka.ms/java-test-config)
```

Comment:
Just to confirm if it's your intention to use the block quotes `>` here? I think the mark is often used to quote something, might not be the best choice here. My personal preference is as following:

#### alternative a
- Redesign ....
  - More details .....

#### alternative b
- Redesign .....   [More details](#)

Reply:
Use the alternative b